src/main/io/osd.c

```diff
@@ -1027,6 +1039,23 @@ static void osdDrawElements(void)
     }
 #endif
 
+#ifdef USE_RTC_TIME
```

Comment:
Looks like something went wrong here with rebasing. All of these (except for the one for blackbox) have been moved to be processed as part of running through `osdElementDisplayOrder`, so they should be removed from here.